### ISSUE TITLE
Won't compile using -pipe option

### ISSUE THREAD

**Issue by muranyia** [ahlstromcj/sequencer64#195]
Tried to compile using the `-pipe` option in order to lessen the use of the SSD, but it failed complaining about a file not found. (Sorry, cannot quote the error as I've lost the log.)

**Comment by ahlstromcj** [comment]
I'm not sure what you mean by the -pipe option.  There's no such option in
seq64's bootstrap/configure.  I see that -pipe is a GNU compiler option, but how
are you getting that option to gcc or g++?

In any case, if you have a fairly new (a couple a years) SSD, I wouldn't worry
about using that option.

However, if you want to explore further and provide more information or requests,
by all means let me know!

Thanks!

-------- muranyia 02:56 Mon 02 Mar      --------

>Tried to compile using the `-pipe` option in order to lessen the use of the SSD, but it failed complaining about a file not found. (Sorry, cannot quote the error as I've lost the log.)
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/ahlstromcj/sequencer64/issues/195

-- 
Imagine if every Thursday your shoes exploded if you tied them the usual
way.  This happens to us all the time with computers, and nobody thinks of
complaining.
		-- Jeff Raskin

**Comment by muranyia** [comment]
I `export CFLAGS` from .bashrc. Basically, I thought I'd speed things up and possibly save some SSD wear&tear as I have plenty of RAM. The process went fine for quite some time but then perhaps a file was explicitly being looked for... Shall I retry and send you the output?

**Comment by ahlstromcj** [comment]
Sure!  It could lead to a good feature for the build!

-------- muranyia 16:17 Sun 22 Mar      --------

>I `export CFLAGS` from .bashrc. Basically, I thought I'd speed things up and possibly save some SSD wear&tear as I have plenty of RAM. The process went fine for quite some time but then perhaps a file was explicitly being looked for... Shall I retry and send you the output?

**Comment by ahlstromcj** [comment]
I was able to get it to build, but I did find a bug in configure.ac that I fixed.  Check out the latest midi_control branch.  Either of these work:

 $ CFLAGS="-pipe" ./bootstrap -ed

 $ CFLAGS="-pipe" ./configure --enable-debug --disable-shared

Adjust for your build options.  Of course, if you have "export CFLAGS="-pipe" in your .bashrc, that would cover all project that you build.

I updated the INSTALL file, too.
